addons/wowl/static/src/core/download.ts

```diff
@@ -0,0 +1,140 @@
+import download from "../libs/download";
+import {_lt, Stringifiable} from "./localization";
```

Comment:
@aab-odoo I used _lt instead or _t, but I'm not sure it's the way to go.

Reply:
I think you need to use "env._t"